__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const testData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("status 200, responds w/ array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const topics = response.body;
+        // expect(typeof topics).toBe(Array); // why does this line not work but line 24 does?
```

Comment:
because Array isn't a "type" in this situation, to do the same thing you would use Array.isArray(topics), however I think this is a bit of an unnecessary test either way